projects/shared/src/lib/component/documents/document-description/document-description.component.html

```
@@ -205,13 +216,33 @@
+                }
+                {{ 'Equinox' | translate }}: {{ cartographic.equinox }}
+              }
             </li>
           }
         </ul>
       </ng-container>
     </shared-description-zone>
   }
 
+  <!-- TEMPORAL COVERAGE -->
+  @if (temporalCoverages.length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)